src/client/SharedQueue.h

```diff
@@ -10,7 +10,7 @@
 #include <thrift/transport/TSocket.h>
 #include <thrift/transport/TTransportUtils.h>
 
-#include "../constants/constants.h"
+#include "constants.h"
```

Comment:
Thoughts on having include statements be formatted as `<library>/header.h`. Makes it easier to find out which library the header comes from and helps avoid name clashing

Reply:
I'd counter that it's bad practice to have files with the same names and you can just `find` or `ctrl + P` to find the file.